components/observations/SimpleForm.tsx

```diff
@@ -28,15 +28,15 @@ import {LoggerContext, LoggerProps} from 'loggerContext';
 import Toast from 'react-native-toast-message';
 import {ObservationsStackNavigationProps} from 'routes';
 import {colorLookup} from 'theme';
-import {AvalancheCenterID, ImageMediaItem, InstabilityDistribution, MediaType, Observation} from 'types/nationalAvalancheCenter';
+import {AvalancheCenterID, ImageMediaItem, InstabilityDistribution, MediaType} from 'types/nationalAvalancheCenter';
 
 export const SimpleForm: React.FC<{
   center_id: AvalancheCenterID;
   onClose?: () => void;
 }> = ({center_id, onClose}) => {
   const navigation = useNavigation<ObservationsStackNavigationProps>();
   const {logger} = React.useContext<LoggerProps>(LoggerContext);
-  const formContext = useForm({
+  const formContext = useForm<ObservationFormData>({
```

Comment:
because defaultValues returns a `Partial<ObservationFormData>`, formContext then decided that it was wrapping that type. this explicitly tells it that no, it's wrapping a full `ObservationFormData` and so that's what methods like `onSubmit` will receive.